libbeat/docs/dashboardsconfig.asciidoc

```diff
@@ -10,7 +10,7 @@
 //////////////////////////////////////////////////////////////////////////
 
 [[configuration-dashboards]]
-== Load the Kibana dashboards
+== Load the Kibana dashboards (deprecated)
```

Comment:
This would mean on our end it shows up as deprecated too? Not sure if we should put this into the title?

Instead you could use `[]deprecated` flag with an if statement around?

Reply:
Sure, I hadn't pushed the latest changes.